src/components/Base/Link/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { translate } from 'react-i18next';
+
+import links from 'config/doc-link';
+
+@translate()
+export default class DocLink extends Component {
```

Comment:
Since the component name is `DocLink`, you should name this file name as `DocLink` too. Also the name is conflict with react-dom `Link`

Reply:
ok